ccd-adapter/src/test/java/uk/gov/hmcts/cmc/ccd/mapper/defendant/ResponseMapperTest.java

```diff
@@ -85,6 +85,19 @@ public void shouldMapFullDefenceResponseFromCCD() {
         assertThat(builder.build().getResponse().orElse(null)).isEqualTo(ccdDefendant);
     }
 
+    @Test
+    public void shouldMapFullDefenceResponseWithFreeMediationFromCCD() {
+        //given
+        CCDDefendant ccdDefendant = SampleCCDDefendant.withFullDefenceResponseAndFreeMediation().build();
+        Claim.ClaimBuilder builder = Claim.builder();
+
+        //when
+        mapper.from(builder, ccdDefendant);
+
+        //then
+        assertThat(builder.build().getResponse().orElse(null)).isEqualTo(ccdDefendant);
```

Comment:
ResponseAssert need to be updated for `MediationPhoneNumber`

Reply:
done